frontend/src/components/MarkdownRender.tsx

```diff
@@ -10,7 +10,15 @@ type MarkdownRenderProps = {
 const MarkdownRender = (props: MarkdownRenderProps) => {
   return (
     <ReactMarkdown
-      allowedTypes={["link", "list", "listItem", "paragraph", "strong", "text"]}
+      allowedTypes={[
+        "link",
+        "list",
+        "listItem",
+        "paragraph",
+        "strong",
+        "text",
+        "heading",
```

Comment:
@denisquesada hi Denis this change is problematic. We should not allow use to enter custom markup. This is will play bad with the heading levels of the app